internal/service/connect/routing_profile.go

```diff
@@ -477,21 +492,51 @@ func expandMediaConcurrencies(tfList []any) []awstypes.MediaConcurrency {
 			Channel:     awstypes.Channel(tfMap["channel"].(string)),
 			Concurrency: aws.Int32(int32(tfMap["concurrency"].(int))),
 		}
+
+		if v, ok := tfMap["cross_channel_behavior"].([]any); ok && len(v) > 0 {
+			crossChannelBehaviorMap := v[0].(map[string]any)
+			apiObject.CrossChannelBehavior = &awstypes.CrossChannelBehavior{
+				BehaviorType: awstypes.BehaviorType(crossChannelBehaviorMap["behavior_type"].(string)),
+			}
+		}
```

Comment:
This can be moved into it's own, distinct expander `expandCrossChannelBehavior`.

Each flatten/expander is typically only responsible for one level of attributes. Because `cross_channel_behavior` is a nested list, we can split that out into it's own expand function which gets called from here. This helps keep the function body of each individual flex function compact, even for large nested structures.